source/MaterialXTest/ShaderValid.cpp

```diff
@@ -603,7 +605,7 @@ TEST_CASE("MaterialX documents", "[shadervalid]")
     #endif
 #endif
 
-    // For debugging, add files to this set to override 
+    // For debugging, add files to this set to override
```

Comment:
Not important. Wondering why you keep having line ending differences in your check-ins?

Reply:
I think Atom is fixing them automagically. I didn't fix them myself. This one clearly had a bad line ending.

Reply:
Maybe it's possible to turn that off in Atom? Not a major problem but it does create a bit of noise in the change diff.

Reply:
It's technically fixing whitespace errors, so it's not the end of the world I feel, but ya, it does create noise.